providers/os/resources/packages/aix_packages.go

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Mondoo, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package packages
+
+import (
+	"bufio"
+	"fmt"
+	"go.mondoo.com/cnquery/providers/os/connection/shared"
+	"io"
+	"strings"
+)
+
+const (
+	AixPkgFormat = "bff"
```

Comment:
Are you going to add RPM packages as well? AIX supports those and it would be very handy to expose those.

Reply:
Hi @scotthain Great idea! Lets do this as follow-up